create-cert-bundle.py

```diff
@@ -0,0 +1,42 @@
+import os
+import logging
+import sys
+
+logger = logging.getLogger()
+handler = logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+if os.environ.get("HTTPS_PROXY") is None:
```

Comment:
Https proxy will not be none for any proxy. But you only need to do this cert stuff when the proxy itself is https.

Reply:
Oh right...my bad. These env vars are set same for any url.I need to check the scheme. Thanks for pointing it out